hugolib/pageSort.go

```diff
@@ -275,3 +276,20 @@ func (p Pages) Reverse() Pages {
 
 	return pages
 }
+
+func (p Pages) ByParam(paramsKeyStr string) Pages {
+	key := "pageSort.ByParam." + paramsKeyStr
+
+	paramsKeyComparator := func(p1, p2 *Page) bool {
+		v1, _ := p1.Param(paramsKeyStr)
```

Comment:
Yes, that was intentional. That way one can provide site-wide defaults. I'll make sure to document that behavior in the follow-up.